2015-05-15 13:45:53,370 [13] Orchard.WebApi.Filters.UnhandledApiExceptionFilter - Default - Unexpected API exception
 http://localhost:30321/api/listing/listing
System.NullReferenceException: Object reference not set to an instance of an object.
   at ivNet.Listing.Service.ListingServices.<>c__DisplayClass7.<GetListings>b__1(ListingDetail x) in f:\Orchard\www.showdigs.co.uk.v.1.8.1\src\Orchard.Web\Modules\ivNet.Listing\Service\ListingServices.cs:line 40
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at ivNet.Listing.Service.ListingServices.GetListings(String eMail) in f:\Orchard\www.showdigs.co.uk.v.1.8.1\src\Orchard.Web\Modules\ivNet.Listing\Service\ListingServices.cs:line 42
   at ivNet.Listing.Controllers.api.ListingController.Get() in f:\Orchard\www.showdigs.co.uk.v.1.8.1\src\Orchard.Web\Modules\ivNet.Listing\Controllers\api\ListingController.cs:line 44
   at lambda_method(Closure , Object , Object[] )
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Orchard.WebApi.Filters.OrchardApiActionFilterDispatcher.<ExecuteActionFilterAsync>d__6.MoveNext() in e:\Orchard\www.showdigs.co.uk.v.1.8.1\src\Orchard\WebApi\Filters\OrchardApiActionFilterDispatcher.cs:line 20
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()
